static-assets/ng-views/settings.html

```diff
@@ -26,7 +26,7 @@ <h1>{{ 'settings.HEADER_LABEL' | translate }}</h1>
             <h2>{{ 'settings.PROFILE' | translate }}</h2>
             <div class="well">
                 <div class="user-display">
-                    <div class="name">{{user.name}} {{user.surname}}</div>
+                    <div class="name">{{user.name || user.first_name}} {{user.surname || user.last_name}}</div>
```

Comment:
Are both property name alternatives in use? If so, when and why two different property names for same thing?

Reply:
What happens is that user data is being set in two scenarios.

1.  No user logged in studio. When there's no user you need to login, and login callback has user data (in this case the prop is `user.name`, for example)

2.  User already logged in. When user is already logged in into studio the user data comes from model, and is set on a variable from the freemarker template, parsed and it's retrieved as the user data (in this scenario the props have different names, like `user.first_name`).